README.md

```diff
@@ -22,10 +22,9 @@ through setting up and using TensorBoard. There's an associated
 
 [tutorial with an end-to-end example of training TensorFlow and using TensorBoard]: https://github.com/dandelionmane/tf-dev-summit-tensorboard-tutorial
 
-# Usage
+#Usage
 
-Before running TensorBoard, make sure you have generated summary data in a log
-directory by creating a summary writer:
+Before running TensorBoard, make sure you have generated summary data or model summary in a log directory by creating a summary writer:
```

Comment:
It is not clear to me whether addition of two words, `model summary`, help here. Can you fill out the Pull Request template and let us know your intent and what you found to be deficient? Thanks!